tezos docker/README.md

```diff
@@ -9,6 +9,11 @@ docker run --mount type=bind,source=<path-to-archive-store>,target=/data tezos_t
 
 where `<path-to-archive-store>` should be a folder containing `archive_store` and the `blocks_above_933913` file. Note that the `archive_store` will be modified by the benchmarks, so be sure that it is a copy of the initial archive store to benchmark (which you can get from `/data/ioana/init_archive_store/` on `comanche`.
 
+In the `offline_bootstrap.sh` script, `tezos_node` connects to a fake baker on port `8732` . The fake baker reads already produced blocks on mainnet, tries to simulate the operations as it would have done if it was normally producing blocks. So, the baker uses `blocks_above_933913` files, and the next block with id `933914` is generated.
```

Comment:
```suggestion
In the `offline_bootstrap.sh` script, `tezos_node` connects to a baker on port `8732`. `tezos-baker-006-PsCARTHA` is a fake baker: it reads file `blocks_above_933913`, which contains already produced blocks on mainnet, and it tries to simulate the operations as it would have done if it was normally producing blocks. So, the baker uses the file to generate the next block with id `933914` and to send it over to the `tezos_node`.
```